src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud.mdx

```diff
@@ -0,0 +1,365 @@
+---
+title: Collector for Confluent Cloud & Kafka monitoring
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - Kafka
+  - Confluent Cloud
+metaDescription: You can collect Kafka metrics from Confluent using the OpenTelemetry collector.
+---
+You can collect metrics about your Confluent Cloud-managed Kafka deployment with the OpenTelemetry collector. The collector is a component of OpenTelemetry that collects, processes, and exports telemetry data to New Relic (or any observability back-end).
+
+<Callout variant="tip">
+  If you're looking for help with other collector use cases, see the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples) repository.
+</Callout>
+
+Here are the steps you'll complete to collect Kafka metrics from Confluent:
```

Comment:
This doc is relatively short, so this real estate might be better used explaining in a general, high level way what processes users can expect (what changes are made to the system and where). If that's too redundant, then I'd just remove this section.

Reply:
@akristen I just removed the mini-TOC and used a simple introductory sentence.